plugins/modules/launchdarkly_feature_flag_environment.py

```diff
@@ -12,66 +12,68 @@
 
 DOCUMENTATION = r"""
 module: launchdarkly_feature_flag_environment
-short_description: Create Environment specific flag targeting
+short_description: Configure feature flag targeting
 description:
-     - Manage LaunchDarkly manage feature flags and account settings.
+     - Configure LaunchDarkly feature flag targeting for a given flag, environment, and project. To learn more, read L(Targeting users with flags, https://docs.launchdarkly.com/home/flags/targeting-users).
```

Comment:
Maybe this should be modified to be something `for a given flag in a specific environment and project`?

Reply:
good call, updated.